shared/api-definitions-ex.asciidoc

```diff
@@ -0,0 +1,30 @@
+[[sample-object]]
+=== Object1 API objects
```

Comment:
```suggestion
=== `Object1` API objects
```

Reply:
I'm not sure how I feel about having formatting in the titles/headings.  I don't think I've seen this other than in the Cloud example.

Reply:
👍 Not a big deal to me.